### ISSUE TITLE
America/Argentina/Buenos_Aires America/Nuuk timezone (probably others) does not return offset, daylight is null

### ISSUE THREAD

**Issue by Romanior** [spencermountain/timezone-soft#13]
"timezone-soft": "^1.3.1",

Just calling calling `soft('America/Argentina/Buenos_Aires')`

returns with:
```
[
    {
        "iana": "America/Argentina/Buenos_Aires",
        "standard": {
            "name": "America/Argentina/Buenos_Aires",
            "abbr": "America/Argentina/Buenos_Aires"
        },
        "daylight": null
    }
]
```

Although calling `soft('America/Argentina/San_Luis')` produces correct result
```
[
    {
        "iana": "America/Argentina/San_Luis",
        "standard": {
            "name": "Argentina Standard Time",
            "abbr": "ART",
            "offset": -3
        },
        "daylight": {}
    }
]
```

Similar bug for `America/Nuuk`

```
{
    "iana": "America/Nuuk",
    "standard": {
        "name": "Nuuk Time",
        "abbr": "UTC-3/-4"
    },
    "daylight": null
}
```

**Comment by spencermountain** [comment]
thanks Roman, this looks like a bug. I'm a bit busy this week, but I'll keep this open - PRs welcome!

**Comment by spencermountain** [comment]
hey Roman, i've added a fix for America/Nuuk in 1.4.0

I believe [America/Argentina/Buenos_Aires and America/Argentina/San_Luis](https://www.timeanddate.com/time/zone/argentina/buenos-aires) both have no dst now - but let me know if you know something I don't!
cheers
